subprojects/testing-jvm/src/main/java/org/gradle/api/internal/tasks/testing/junitplatform/JUnitPlatformTestFramework.java

```diff
@@ -57,9 +58,23 @@ public JUnitPlatformTestFramework(DefaultTestFilter filter) {
     }
 
     public JUnitPlatformTestFramework(DefaultTestFilter filter, boolean useImplementationDependencies) {
+        this(filter, useImplementationDependencies, new JUnitPlatformOptions());
```

Comment:
I think that the constructor above in line 56 can we removed once the test retry plugin can be built against Gradle 8 with a new nightly version.